src/includes/getting-started-primer/java.mdx

```diff
@@ -0,0 +1,13 @@
+<Note>
+<markdown>
+
+_Sentry's Java SDK enables reporting messages and errors._
+
+Sentry for Java is a collection of modules provided by Sentry. At its core, Sentry for Java provides a raw client for sending events to Sentry. To begin, we **highly recommend** you use one of the logging libraries or framework integrations. We also provide documentation for [manual usage](./usage/)
+
+</markdown>
+</Note>
+
+<!-- TODO: add version and value information, similar to what we have for Go:
```

Comment:
Good point!

We can state we support Java 1.8 and later here

Reply:
Great. I will add that. Is there anything else we'd like to add?